docs/sources/reference/commandline/cli.md

```diff
@@ -197,30 +201,31 @@ To run the daemon with debug output, use `docker -d -D`.
 The Docker daemon can listen for [Docker Remote API](/reference/api/docker_remote_api/)
 requests via three different types of Socket: `unix`, `tcp`, and `fd`.
 
-By default, a `unix` domain socket (or IPC socket) is created at `/var/run/docker.sock`,
-requiring either `root` permission, or `docker` group membership.
+By default, a `unix` domain socket (or IPC socket) is created at
+`/var/run/docker.sock`, requiring either `root` permission, or `docker` group
+membership.
 
 If you need to access the Docker daemon remotely, you need to enable the `tcp`
-Socket. Beware that the default setup provides un-encrypted and un-authenticated
-direct access to the Docker daemon - and should be secured either using the
-[built in HTTPS encrypted socket](/articles/https/), or by putting a secure web
-proxy in front of it. You can listen on port `2375` on all network interfaces
-with `-H tcp://0.0.0.0:2375`, or on a particular network interface using its IP
-address: `-H tcp://192.168.59.103:2375`. It is conventional to use port `2375`
-for un-encrypted, and port `2376` for encrypted communication with the daemon.
-
-> **Note** If you're using an HTTPS encrypted socket, keep in mind that only TLS1.0
-> and greater are supported. Protocols SSLv3 and under are not supported anymore
-> for security reasons.
+Socket. Beware that the default setup provides un-encrypted and
+un-authenticated direct access to the Docker daemon - and should be secured
+either using the [built in HTTPS encrypted socket](/articles/https/), or by
+putting a secure web proxy in front of it. You can listen on port `2375` on all
+network interfaces with `-H tcp://0.0.0.0:2375`, or on a particular network
+interface using its IP address: `-H tcp://192.168.59.103:2375`. It is
+conventional to use port `2375` for un-encrypted, and port `2376` for encrypted
+communication with the daemon.
+
+> **Note** If you're using an HTTPS encrypted socket, keep in mind that only
```

Comment:
Do we want the colon in the bold? I think yes, but just now noticed that is inconsistent.